modules/services/pueue.nix

```diff
@@ -49,7 +51,7 @@ in {
 
         Service = {
           Restart = "on-failure";
-          ExecStart = "${cfg.package}/bin/pueued -v -c ${configFile}";
+          ExecStart = "${cfg.package}/bin/pueued -v";
```

Comment:
The configuration file should still be defined here, otherwise we won't know when the service needs to be restarted on configuration changes.